### PR TITLE
update import user instructions

### PR DESCRIPTION
**Pull request by Ephraim-G**
## Changes proposed in this pull request:
- update instructions on how to format the google sheet for user import into zendesk

## security considerations
N/A

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/pages-3-zendesk-sync.md [hunk 1]
```diff
@@ -32,14 +32,34 @@ Monthly review to make sure cloud.gov Pages users, orgs, and roles are accuratel
 
 ### User Import
 
-- Download and open the CSV export in Google Sheets for editing purposes
-- Delete all columns besides "Email" "Organizations" "Name" "Details"
+- Download and Import the Users CSV into Google Sheets for editing purposes
+- For "Sperator type" keep it unchanged from "Detect automatically"
```
Comment (svenaas) [inline .github/ISSUE_TEMPLATE/pages-3-zendesk-sync.md:36]:
```suggestion
- For "Seperator type" keep it unchanged from "Detect automatically" 
```

File: .github/ISSUE_TEMPLATE/pages-3-zendesk-sync.md [hunk 2]
```diff
@@ -32,14 +32,34 @@ Monthly review to make sure cloud.gov Pages users, orgs, and roles are accuratel
 
 ### User Import
 
-- Download and open the CSV export in Google Sheets for editing purposes
-- Delete all columns besides "Email" "Organizations" "Name" "Details"
+- Download and Import the Users CSV into Google Sheets for editing purposes
+- For "Sperator type" keep it unchanged from "Detect automatically" 
+- Delete the columns "Created" and "Last Signed In"
+- Delete all of the content in "ID"
+- The remaning colums should be "ID", "Email", "Organizations" and "Details"
```
Comment (svenaas) [inline .github/ISSUE_TEMPLATE/pages-3-zendesk-sync.md:39]:
```suggestion
- The remaining columns should be "ID", "Email", "Organizations" and "Details"
```

File: .github/ISSUE_TEMPLATE/pages-3-zendesk-sync.md [hunk 3]
```diff
@@ -32,14 +32,34 @@ Monthly review to make sure cloud.gov Pages users, orgs, and roles are accuratel
...
 *The "Name" and "Email" fields are required for importing new users, without them the entire job will fail*
-- Save the sheet as an CSV
+
+- Copy and paste the entire "Email" column into the blank column (A)
```
Comment (svenaas) [inline .github/ISSUE_TEMPLATE/pages-3-zendesk-sync.md:42]: Does this replace the "ID" column?

File: .github/ISSUE_TEMPLATE/pages-3-zendesk-sync.md [hunk 4]
```diff
@@ -32,14 +32,34 @@ Monthly review to make sure cloud.gov Pages users, orgs, and roles are accuratel
...
+- Rename the header to "Name" from "Email"
+- Utilize the "Find and replace" function by hitting command + shift + H
+- In search choose "Specific range" from the dropdown and specify A:A as the range parameter
+- In "Find" specify all email addresses that need to be replaced and leave "Replaced with" null
+(ie. Find:@gsa.gov Replace with: )
+- This will remove all the email address from the names under the "Name" header making it easier to format for importing purposes
+- Replace all "." with an " ," there is a space then comma
```
Comment (svenaas) [inline .github/ISSUE_TEMPLATE/pages-3-zendesk-sync.md:49]:
```suggestion
- Replace all "." with an " ," — that is, a space then a comma
```

File: .github/ISSUE_TEMPLATE/pages-3-zendesk-sync.md [hunk 5]
```diff
@@ -32,14 +32,34 @@ Monthly review to make sure cloud.gov Pages users, orgs, and roles are accuratel
...
+(ie. pages.user -> pages, user)
```
Comment (svenaas) [inline .github/ISSUE_TEMPLATE/pages-3-zendesk-sync.md:50]:
```suggestion
(i.e. from "pages.user" to "pages, user")
```

File: .github/ISSUE_TEMPLATE/pages-3-zendesk-sync.md [hunk 6]
```diff
@@ -32,14 +32,34 @@ Monthly review to make sure cloud.gov Pages users, orgs, and roles are accuratel
...
+- The remaning colums should be "ID", "Email", "Organizations" and "Details"
 *The "Name" and "Email" fields are required for importing new users, without them the entire job will fail*
-- Save the sheet as an CSV
+
+- Copy and paste the entire "Email" column into the blank column (A)
+- Rename the header to "Name" from "Email"
+- Utilize the "Find and replace" function by hitting command + shift + H
+- In search choose "Specific range" from the dropdown and specify A:A as the range parameter
+- In "Find" specify all email addresses that need to be replaced and leave "Replaced with" null
+(ie. Find:@gsa.gov Replace with: )
```
Comment (svenaas) [inline .github/ISSUE_TEMPLATE/pages-3-zendesk-sync.md:47]:
```suggestion
(i.e. Find: "@gsa.gov" Replace with: "")
```

File: .github/ISSUE_TEMPLATE/pages-3-zendesk-sync.md [hunk 7]
```diff
@@ -32,14 +32,34 @@ Monthly review to make sure cloud.gov Pages users, orgs, and roles are accuratel
...
+- This will remove all the email address from the names under the "Name" header making it easier to format for importing purposes
+- Replace all "." with an " ," there is a space then comma
+(ie. pages.user -> pages, user)
+- Save the sheet as a CSV
 - Navigate to the Zendesk admin center 
 - In the lefthand panel locate "Bulk actions" and choose "Import organizations"
 - An import job report will be sent to your .gov inbox where you can view if the job was successful/unsuccesful
```
Comment (svenaas) [inline .github/ISSUE_TEMPLATE/pages-3-zendesk-sync.md:54]:
```suggestion
- An import job report will be sent to your .gov inbox where you can view if the job was successful/unsuccessful
```

File: .github/ISSUE_TEMPLATE/pages-3-zendesk-sync.md [hunk 8]
```diff
@@ -32,14 +32,34 @@ Monthly review to make sure cloud.gov Pages users, orgs, and roles are accuratel
...
 
+**Note** In the csv you will notice a number of users emails towards the bottom which do not have the organization or details field associated with them and will consequently result as failures in the report results as such
```
Comment (svenaas) [inline .github/ISSUE_TEMPLATE/pages-3-zendesk-sync.md:56]:
```suggestion
**Note** In the CSV you will notice a number of users' emails towards the bottom which do not have the organization or details field associated with them and will consequently result as failures in the report results. For example:
```

**Review by svenaas — CHANGES_REQUESTED** [review]
This is great; thanks @Ephraim-G. I've made a bunch of little copy suggestions and asked one question that might inspire a minor edit.

File: .github/ISSUE_TEMPLATE/pages-3-zendesk-sync.md (hunk 3 above)
Comment (Ephraim-G) [reply .github/ISSUE_TEMPLATE/pages-3-zendesk-sync.md:42]: Yep just replacing everything in that column.

Comment (svenaas) [reply .github/ISSUE_TEMPLATE/pages-3-zendesk-sync.md:42]: Maybe 

```suggestion
- Copy and paste the entire "Email" column into the blank column (A) that used to be the "ID" column
```

for clarity?

**Comment by Ephraim-G** [comment]
@svenaas alright good to go 👍🏽

**Review by svenaas — APPROVED** [review]
LGTM!

**Pull request merged**